<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTodoItemDialog</class>
 <widget class="QDialog" name="AddTodoItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add TODO item</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>221</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="todoName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>TODO name (required)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="estimatedPomodoros">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Estimated Pomodoros</string>
   </property>
  </widget>
  <widget class="QComboBox" name="tags">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Tags</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTags">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quick add tag</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>todoName</tabstop>
  <tabstop>estimatedPomodoros</tabstop>
  <tabstop>leTags</tabstop>
  <tabstop>tags</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTodoItemDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTodoItemDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
